### ISSUE TITLE
[Potential Bug] .selectedIndexes() crashing

### ISSUE THREAD

**Issue by Bluscream** [pathmann/PythonQt#1]
Seems like in the connection of a QTableWidget to the `def on_table_doubleClicked(self):`
crashes when trying to get `self.table.selectedIndexes()`

After trying to debug it further i think it's a pythonQt bug, can you test if it works for you?

The parts triggering the crash:

```python
class ServersDialog(QDialog):
    def __init__(self,serverBrowser, parent=None):
        self.serverBrowser=serverBrowser
        super(QDialog, self).__init__(parent)
        setupUi(self, os.path.join(ts3.getPluginPath(), "pyTSon", "scripts", "serverBrowser", "ui", "servers.ui"), self.CONF_WIDGETS)
        self.setupList()
        self.setWindowTitle("PlanetTeamspeak Server Browser")
        #try:
            #self.serverList.doubleClicked.connect(self.table_doubleclicked)
            #self.serverList.connect("doubleClicked()", self.table_doubleclicked)
            #self.apply.connect("clicked()", self.on_apply_clicked)
            #self.reload.connect("clicked()", self.on_reload_clicked)
        #except:
            #ts3.logMessage(traceback.format_exc(), ts3defines.LogLevel.LogLevel_ERROR, "PyTSon", 0)

    def on_serverList_doubleClicked(self):
        try:
            ts3.logMessage("test", ts3defines.LogLevel.LogLevel_INFO, "pyTSon", 0)
            index = self.serverList.selectedIndexes()[0]
            ts3.logMessage("test2", ts3defines.LogLevel.LogLevel_INFO, "pyTSon", 0)
            # id_us = int(self.serverList.model().data(index).toString())
            # ts3.printMessageToCurrentTab("index : " + str(id_us))
        except:
            ts3.logMessage(traceback.format_exc(), ts3defines.LogLevel.LogLevel_ERROR, "pyTSon", 0)
```
https://github.com/Bluscream/betterServerBrowser/blob/master/plugins/pyTSon/scripts/serverBrowser.py#L333

**Comment by Bluscream** [comment]
https://github.com/Bluscream/betterServerBrowser/blob/master/plugins/pyTSon/scripts/serverBrowser/ui/servers.ui#L48

``` xml
      <widget class="QTableWidget" name="serverList">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>800</width>
         <height>0</height>
        </size>
       </property>
       <property name="autoScroll">
        <bool>true</bool>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::SingleSelection</enum>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
       <property name="showGrid">
        <bool>false</bool>
       </property>
       <property name="sortingEnabled">
        <bool>true</bool>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
       <property name="columnCount">
        <number>6</number>
       </property>
       <attribute name="horizontalHeaderVisible">
        <bool>true</bool>
       </attribute>
       <attribute name="horizontalHeaderDefaultSectionSize">
        <number>100</number>
       </attribute>
       <attribute name="horizontalHeaderMinimumSectionSize">
        <number>0</number>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>true</bool>
       </attribute>
       <column>
        <property name="text">
         <string>Premium</string>
        </property>
        <property name="toolTip">
         <string>Premium</string>
        </property>
        <property name="font">
         <font>
          <weight>50</weight>
          <bold>false</bold>
          <kerning>true</kerning>
         </font>
        </property>
        <property name="textAlignment">
         <set>AlignCenter</set>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Name</string>
        </property>
        <property name="toolTip">
         <string>Server Name</string>
        </property>
        <property name="textAlignment">
         <set>AlignCenter</set>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Clients</string>
        </property>
        <property name="toolTip">
         <string>Clients / Slots</string>
        </property>
        <property name="textAlignment">
         <set>AlignCenter</set>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Country</string>
        </property>
        <property name="toolTip">
         <string>Country</string>
        </property>
        <property name="textAlignment">
         <set>AlignCenter</set>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Create Channel</string>
        </property>
        <property name="toolTip">
         <string>Everyone can create a channel on this server</string>
        </property>
        <property name="textAlignment">
         <set>AlignCenter</set>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Password</string>
        </property>
        <property name="toolTip">
         <string>This server is password protected</string>
        </property>
        <property name="textAlignment">
         <set>AlignCenter</set>
        </property>
       </column>
      </widget>
```

**Comment by pathmann** [comment]
EDIT: OK, missread bug/title.

I'll have a look into it.

But it's good practice to provide a running (running, but triggers the bug) example (eg. without dependencies).

**Comment by pathmann** [comment]
But keep in mind that you are mixing QTableView- and QTableWidget-related functions/signals here!
And btw. selectedIndexes is virtual proteced!

The itemDoubleClicked-signal of QTableWidget is what you are looking for.

**Comment by Bluscream** [comment]
@pathmann ist svens server down?

**Comment by pathmann** [comment]
Closed in favor of #2.
